### PR TITLE
L2-727: Remove redirections to Flutter

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

Remove redirections from svelte to flutter.

# Changes

* Remove usage of `REDIRECT_TO_LEGACY` in svelte app.
* Remove all `SHOW_XXX` constants.
* Remove all checks with `SHOW_XXX` onMount to redirect to flutter.

# Tests

Not applicable.

### CODE REVIEW

**Comment by lmuntaner** [comment]
@peterpeterparker @bitdivine please review

File: frontend/svelte/env.config.mjs [hunk 1]
```diff
@@ -69,8 +69,6 @@ const LEDGER_CANISTER_URL = getRequiredEnvVar("LEDGER_CANISTER_URL");
 const OWN_CANISTER_URL = getRequiredEnvVar("OWN_CANISTER_URL");
 
 // Configuration
-// ... Redirects between the svelte and flutter frontends
-const REDIRECT_TO_LEGACY = getRequiredEnvVar("REDIRECT_TO_LEGACY");
```
Comment (peterpeterparker) [inline frontend/svelte/env.config.mjs:73]: should we fully remove it form the config or have we gonna use it to obfuscate sns features?

Comment (lmuntaner) [reply frontend/svelte/env.config.mjs:73]: We might want to use feature flags for SNS, I agree, but I would use the DEPLOY_ENV for that and use flag specifically for that.

This env var us used from the build, to flutter, across all the deployment and build process. I believe that it's easier to totally remove it from the pipeline.

Comment (lmuntaner) [reply frontend/svelte/env.config.mjs:73]: @bitdivine what do you think? Do you want to use this env var for SNS?

File: frontend/svelte/src/routes/Neurons.svelte [hunk 2]
```diff
@@ -54,70 +46,65 @@
...
+          {neuron}
+        />
+      {/each}
+    {/if}
+  </section>
+  <svelte:fragment slot="footer">
+    <Toolbar>
+      <button
+        data-tid="stake-neuron-button"
+        class="primary"
```
Comment (bitdivine) [inline frontend/svelte/src/routes/Neurons.svelte:77]: I think this is unrelated to removing redirects?  I have no idea whether it's good or bad or intentional but it looks unrelated.  Same below.

**Review by bitdivine — APPROVED** [review]
There a couple of small changes that aren't clearly related to removing redirects.  Are these intentional?

File: frontend/svelte/src/routes/Neurons.svelte (hunk 2 above)
Comment (lmuntaner) [reply frontend/svelte/src/routes/Neurons.svelte:77]: oh, it should be full-width, good point. This happened during a conflict fix, thanks!

File: frontend/svelte/env.config.mjs (hunk 1 above)
Comment (bitdivine) [reply frontend/svelte/env.config.mjs:73]: I had been thinking of it but at minimum we'd want to rename it.  A dedicated feature flag `SHOW_SNS` would work, and yes, we could enable that for some deployment targets in dfx.json

File: frontend/svelte/src/routes/Neurons.svelte (hunk 2 above)
Comment (lmuntaner) [reply frontend/svelte/src/routes/Neurons.svelte]: fixed, I will do a quick recheck of all the main pages before merging the PR.

File: frontend/svelte/env.config.mjs (hunk 1 above)
Comment (bitdivine) [reply frontend/svelte/env.config.mjs:73]: Personally I'm leaning more towards a feature flag.  What do you think?

Comment (lmuntaner) [reply frontend/svelte/env.config.mjs:73]: Yes, I agree.
